vendor/wp-now/src/wp-now.ts

```diff
@@ -176,6 +179,39 @@ async function prepareWordPress( php: PHP, options: WPNowOptions ) {
 			await runWpPlaygroundMode(php, options);
 			break;
 	}
+
+	// Symlink manager is not yet supported on windows
+	// See: https://github.com/Automattic/studio/issues/548
+	if ( !process.platform.startsWith('win') ) {
```

Comment:
In other areas of the app, we check if it's running on the Windows platform with the following code:

```suggestion
	if ( process.platform !== 'win32' ) {
```

Reply:
I saw that and it made me wonder if at some stage it could potentially be win64?  For now I'll change it to keep it in line with the rest.

Reply:
Yeah, good point. The `platform` value is provided by Node.js and there's no `win64` value ([reference](https://nodejs.org/api/process.html#processplatform)). I think even when using Windows 64 bits it references `win32`.

Reply:
Win32 is the name of the platform API and not the processor architecture.

Reply:
Thats good to know, thanks for the info!